domain.md

```diff
@@ -0,0 +1,71 @@
+ * In the **Permissions** page, click to select the following permission check boxes from list:  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Reset Password**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Read and write Account Restrictions**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to DNS host name**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to service principal name**
+ * Click **Next** to **the Completing the Delegation of Control Wizard** page, after review then click **Finish**.
+
+
+# 2) Enable remote powershell admin on a Domain Controller (DC) machine. 
+
+The following steps enable remote powershell admin
+
+* Login Domain Controller Server.
+* Start Windows PowerShell as an administrator.
+* Check the configuration of WinRM service by running following command:  
+  **Get-WmiObject Win32_Service | Where-Object { $_.Name -eq 'WinRM'} | Format-Table**  
+  The value of the StartMode property in the output should be "Auto" and the value of the Status property in the output should be "Running"
+* Set the StartMode property of WinRM service to Automatic by running following command if needed:  
+  **Set-Service WinRM -StartupType Automatic**  
+  **Restart-Service -Name WinRM**
+* run command to enable remote powershell: **Enable-PSRemoting -Force**
+
+
+# 3) Enable LDAPs on a on Domain Controller (DC) machine. 
+    
+Basically, there are two methods of enabling LDAPS on a DC.
+
+* Method #1: install an Enterprise Root CA  
+  The first method is the easiest: LDAPS is automatically enabled when you install an Enterprise Root CA on a Domain Controller. If you install the AD-CS role and specify the type of setup as ��Enterprise�� on a DC, all DCs in the forest will be automatically be configured to accept LDAPS.  
+    
+  
+* Method #2: add a Digital Certificate on DC  
+  Requirements for an LDAPS certificate:  
+  1) Digital Certificate must be valid for the purpose of ��Server Authentication.�� This means that they must contain the Server Authentication object identifier (OID). OIDs are like the Internet domain name space. They are series of numbers separated by dots, each with a specific meaning. For this purpose, the relevant OID we��re looking for is 1.3.6.1.5.5.7.3.1.  
+  2) The Subject name or the first name in the Subject Alternative Name (SAN) must match the Fully Qualified Domain Name (FQDN) of the host machine, such as Subject:CN=server.domain.com.  
+  3) The host machine account needs to have access to the private key. This is done when the digital certificate request is issued from that machine, or when the private key was exported and imported to a different machine.  
+    
+  The following steps setup LDAPS:  
+  1)  Create the request file.
```

Comment:
How do you do this?

Reply:
there are different tool to create certificate request file.  Windows has certreq tool, Java has keytool

Reply:
"Create the certificate request (usually PEM) file with your usual certificate generation tool." ? Something like that?

Reply:
yes, 
Java: https://docs.oracle.com/cd/E19509-01/820-3503/ggezu/index.html
openssl: https://www.ssl.com/how-to/manually-generate-a-certificate-signing-request-csr-using-openssl/
https://blogs.technet.microsoft.com/pki/2009/08/05/how-to-create-a-web-server-ssl-certificate-manually/